website/docs/faqs/Accounts/change-users-license.md

```diff
@@ -0,0 +1,19 @@
+---
+title: How do I change a users license type to read-only in dbt Cloud?
+description: "Changing a users license type to read-only in dbt Cloud"
+sidebar_label: 'How to change a users license type to read-only'
+id: change-users-license
+
+---
+
+You must be an account owner/admin to perform these actions.
+
+1. From the cloud console, navigate to the settings icon and select **Account Settings**
+
+<Lightbox src="/img/docs/dbt-cloud/Navigate To Account Settings.png" title="Navigate to account settings" />
+
+2. Navigate to the left side menu and select **Users**
+3. Select the desired user and choose **Edit** from the bottom of their profile pane
+4. Change the `License Type` from `Developer` to `Read-only` and then select **Save**
```

Comment:
I would only use single quotes for code. We can change this to quotes and I think we can help them find the section just by naming the title "License."

```suggestion
4. Under License, change from "Developer" to "Read-only", and click **Save**.
```